plexpy/notifiers.py

```diff
@@ -1417,3 +1437,68 @@ def return_config_options(self):
                          ]
 
         return config_option
+
+class TELEGRAM(object):
+
+    def __init__(self):
+        self.enabled = plexpy.CONFIG.TELEGRAM_ENABLED
+        self.bot_token = plexpy.CONFIG.TELEGRAM_BOT_TOKEN
+        self.chat_id = plexpy.CONFIG.TELEGRAM_CHAT_ID
+
+    def conf(self, options):
+        return cherrypy.config['config'].get('Telegram', options)
+
+    def notify(self, message, event):
+        if not message or not event:
+            return
+
+        http_handler = HTTPSConnection("api.telegram.org")
+
+        data = {'chat_id': self.chat_id,
+                'text': message.encode("utf-8")}
```

Comment:
We're dropping the message subject here, can't we do something like with the Twitter notifications and format it like:

Subject: Message